jekyll/_cci2/overview.md

```diff
@@ -0,0 +1,128 @@
+
+## Customer Use Cases 
+
+**Coinbase** runs CircleCI behind their firewall for security and reliability. The [Coinbase case study report](https://circleci.com/customers/coinbase/) reveals that using CircleCI reduced their average time from merge to deploy in half, reduced operations time spent on continuous integration maintenance from 50% of one person's time to less than one hour per week, and increased developer throughput by 20%. 
+
+**Fanatics** increased their team efficiency by 300% with CircleCI. See the [Fanatics case study report](https://circleci.com/customers/fanatics/).
+
+CircleCI enables **Cruise Automation** (a subsidiary of GM) to run many more simulations before putting code in a road test, see the [Cruise case study report](https://circleci.com/customers/cruise/).
+
+Refer to the [CircleCI Customers page](https://circleci.com/customers/) for the complete set of case studies for companies large and small who are using CircleCI.
+
+## Basic Features
+
+After a software repository in GitHub or GitHub Enterprise is added as a project to the CircleCI application, every new commit triggers a build and notification of success or failure through webhooks with integrations for Slack, HipChat, Campfire, Flowdock, or IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
+
+CircleCI may also be configured to deploy code to various environments, including the following:
+
+- AWS CodeDeploy
+- AWS EC2 Container Service (ECS)
+- AWS S3
+- Google Container Engine (GKE)
+- Heroku
+
+Other cloud service deployments can be easily scripted using SSH or by installing the API client of the service with your job configuration.
+
+Contact us at <enterprise-support@circleci.com> to request a trial license.
+
+## Build Environments
+
+By default, CircleCI comes with a general-purpose image based on Ubuntu 14.04 (Trusty). It is possible to customize this image if necessary. 
+
+## Architecture
+
+CircleCI consists of two primary components: Services and Builders. Services run on a single instance that is comprised of the core application, storage, and networking functionality. Any number of Builder instances execute your jobs and communicate back to the Services. Both components must access your instance of GitHub or GitHub Enterprise on the network as illustrated in the following architecture diagram.
+
+![A Diagram of the CircleCI Architecture]({{site.baseurl}}/assets/img/docs/ccie_arch.png)
+
+### Services 
+
+The machine on which the Service instance runs must not be restarted and may be backed up using built-in VM snapshotting. **Note:** It is possible to configure external data storage with PostgreSQL and Mongo for high availability and then use standard tooling for database backups. DNS resolution must point to the IP address of the machine on which the Services are installed. The following table describes the ports used for traffic on the Service instance:
```

Comment:
Consider linking "high availability" to that page in the docs.

Reply:
thanks, added link